javagen/src/main/java/com/azure/autorest/mapper/ClientMethodMapper.java

```diff
@@ -1013,6 +1015,12 @@ protected JavaVisibility methodVisibility(ClientMethodType methodType, boolean h
                     : NOT_GENERATE;
             }
         } else {
+            // Don't generate PagedFlux or PagedIterable methods without Context if Context methods are being generated.
+            if ((methodType == ClientMethodType.PagingSync || methodType == ClientMethodType.PagingAsync)
+                && !hasContextParameter && settings.isContextClientMethodParameter()) {
+                return NOT_GENERATE;
+            }
```

Comment:
This actually is wrong. The overload of required only parameters still need to be generated.

Reply:
Thanks for catching that and adding tests. `ClientMethodMapper` and a few other locations definitely need more unit testing before we begin separating out configurations in `javagen` to the other generators like `fluentgen`